dex/src/matching.rs

```diff
@@ -16,12 +16,25 @@ use crate::{
     state::{Event, EventQueue, EventView, MarketState, OpenOrders, RequestView},
 };
 
+use bytemuck::{cast};
+
 #[cfg(not(feature = "program"))]
 macro_rules! msg {
     ($($i:expr),*) => { { ($($i),*) } };
 }
 declare_check_assert_macros!(SourceFileId::Matching);
 
+pub trait ToAlignedBytes {
```

Comment:
Where is this trait used?

Reply:
line 520, 
`solana_program::system_program::ID.to_aligned_bytes()`